server/client.go

```diff
@@ -161,6 +166,27 @@ func (client JiraClient) RESTPostAttachment(issueID string, data []byte, name st
 	return attachments[0], nil
 }
 
+func (client JiraClient) GetLabels(value, url string) (LabelResult, error) {
+	apiEndpoint := fmt.Sprintf("%s/rest/api/2/jql/autocompletedata/suggestions", url)
+	req, err := client.Jira.NewRequest("GET", apiEndpoint, nil)
+	if err != nil {
+		return LabelResult{}, err
+	}
+
+	q := req.URL.Query()
+	q.Add("fieldName", "labels")
```

Comment:
What are the options for the `fieldName` here? Could we search for a custom label type, or potentially a user/assignee? That would be great if the code is basically identical across all of the autocomplete endpoints.

Reply:
I made this change